src/graph/rg_matrix/rg_matrix.c

```diff
@@ -244,3 +244,59 @@ void RG_Matrix_validateState
 }
 
 
+void RG_Matrix_get
+(
+	RG_Matrix C,
+	GrB_Matrix M
+) {
+	ASSERT(C != NULL);
+	ASSERT(A != NULL);
+
+	GrB_Info    info         =  GrB_SUCCESS;
+	GrB_Matrix  m            =  RG_MATRIX_M(C);
+	GrB_Matrix  delta_plus   =  RG_MATRIX_DELTA_PLUS(C);
+	GrB_Matrix  delta_minus  =  RG_MATRIX_DELTA_MINUS(C);
+
+	info = GrB_Matrix_dup(&M, m);
```

Comment:
Consider using `GrB_transpose`, given that the output matrix been allocated.
```
GrB_transpose(M, dm, NULL, m, GrB_DESC_RSCT0);
```
As `T(T(m)) = m`
this will perform both duplication and masking at once
Also consider renaming `M` to `A` as this is now confusing

Need to check which of the two methods is faster